Modules/Segmentation/SignedDistanceFunction/wrapping/itkPCAShapeSignedDistanceFunction.wrap

```diff
@@ -8,6 +8,7 @@ foreach(d ${ITK_WRAP_IMAGE_DIMS})
   # `double` because of OptimizerParameters is hardcoded as templated
   # over `double` in parent class `ShapeSignedDistanceFunction`.
   foreach(t ${types})
+    string(APPEND ITK_WRAP_PYTHON_SWIG_EXT "DECL_PYTHON_STD_VEC_RAW_TO_SMARTPTR_TYPEMAP(itkImage${ITKM_${t}}${d}, itkImage${ITKM_${t}}${d}_Pointer)\n")
```

Comment:
Is this expected to work in remote modules? Should I replace [the old way](https://github.com/dzenanz/ANTsWasm/commit/f7e72cf5bb2a11837cb18aa13166cd2a56652221) with this style?

Reply:
Yes